src/main/java/org/broadinstitute/hellbender/utils/read/GoogleGenomicsReadToGATKReadAdapter.java

```diff
@@ -517,7 +518,7 @@ private String getRawAttributeValue( final String attributeName, final String ta
             throw new GATKException.ReadAttributeTypeMismatch(attributeName, targetType);
         }
 
-        return rawValue.get(0);
+        return (String)rawValue.get(0);
```

Comment:
add a comment why this cast will succeed (ie link to https://developers.google.com/resources/api-libraries/documentation/genomics/v1/java/latest/com/google/api/services/genomics/model/Read.html#setInfo(java.util.Map). The API is silly (`List<Object>` storing `String`) so we should put the blame for ugly code where it belongs.